src/runtime.js

```diff
@@ -71,6 +70,8 @@ import {waitForBody} from './dom';
 import * as config from './config';
 
 initLogConstructor();
+dev();
+user();
```

Comment:
Forgot to send this draft from yesterday 😄. I think @cramforce already sent similar comments. So ignore if fixed.